Blogplace.Web/Program.cs

```diff
@@ -44,7 +48,7 @@
     app.UseAuthentication();
     app.UseAuthorization();
 
-    app.MapCustomException();
+    app.UseExceptionHandler(_ => { });
```

Comment:
_ => {}
to ma tak być?

Reply:
Tak, już sprawdzałem na kilka sposobów. Jak nie ma tego delegate'a to nic nie działa już przy starcie.